models/section.schema.js

```diff
@@ -0,0 +1,14 @@
+import segmentSchema from './segment.schema';
```

Comment:
🚫 **[eslint]** <[import/newline-after-import](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/newline-after-import.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected 1 empty line after import statement not followed by another import.
```suggestion
import segmentSchema from './segment.schema';

```